Automattic/wp-calypso
# SiteIconPicker: add guard for empty image file

Fix for logged error reported [here](p1688549957486639-slack-C04U5A26MJB) 

## Proposed Changes

* Add a guard to check the file size before accepting it as valid

## Testing Instructions
- Visit http://calypso.localhost:3000/setup/free-post-setup/freePostSetup?siteSlug=your-site-url
- Upload an empty png file. You can create one with `touch empty.png`
- No actions should get triggered and no error should be thrown.

### Bad file type test
- Rename any other image to png. In my case I used a video.
- The modal should close and no errors should be thrown.

**Current behavior**
https://github.com/Automattic/wp-calypso/assets/7000684/6af437da-ea15-4ecd-8bb9-c480a00cfe53



## Pre-merge Checklist
